superset/assets/src/visualizations/nvd3/NVD3Vis.js

```diff
@@ -340,6 +340,26 @@ function nvd3Vis(element, props) {
         }
         chart.lines.dispatch.on('elementClick', function (e) {
           if (tableFilter) {
+            const removeSelection = (selection && selection.seriesIndex == e.seriesIndex && selection.pointIndex == e.pointIndex);
```

Comment:
We can move this line more close to place where we need this. Currently it is creation . confusion with highlight

Reply:
done